tools/CoherenceBuild/CoherenceBuild.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp1.0</TargetFramework>
     <OutputType>Exe</OutputType>
+    <RuntimeFrameworkVersion>1.0.5</RuntimeFrameworkVersion>
```

Comment:
Do you know what's still using the old runtime though? Apart from the executable project?